pyiron_atomistics/lammps/potentials.py

```diff
@@ -358,11 +361,21 @@ def _get_scale(config):
                 )
         return
 
-    def __init__(self, *chemical_elements, name=None, pair_style=None):
+    def __init__(
+            self,
+            *chemical_elements: str,
+            choice: int = 0,
```

Comment:
Not sure if this should be preset. I would write `choice: Optional[int] = None`, so that the warning can be safely silenced when `choice=0` is explicitly chosen.

Reply:
That's a good improvement. Done.